tests/unit/containers/admin-portal/AdminPortal.spec.jsx

```diff
@@ -0,0 +1,53 @@
+import { configureStore } from '@reduxjs/toolkit'
+import reducer from '~/redux/reducer'
+import {
+  createMemoryRouter,
+  createRoutesFromElements,
+  RouterProvider
+} from 'react-router-dom'
+import { routerConfig } from '~/router/router'
+import { render, screen } from '@testing-library/react'
+import { Provider } from 'react-redux'
+import { ThemeProvider } from '@mui/material/styles'
+import { theme } from '~/styles/app-theme/custom-mui.styles'
+import { beforeEach, expect } from 'vitest'
+
+import AdminPortal from '~/containers/layout/admin-portal/AdminPortal'
+
+const renderAdminPortalRouter = (initialEntries, preloadedState) => {
+  const store = configureStore({
+    reducer: { appMain: reducer },
+    preloadedState
+  })
+  const router = createMemoryRouter(createRoutesFromElements(routerConfig), {
+    initialEntries: [initialEntries]
+  })
+
+  render(
+    <Provider store={store}>
+      <ThemeProvider theme={theme}>
+        <RouterProvider router={router}>
```

Comment:
try to use renderWithProviders instead of these providers

Reply:
I have tried to use renderWithProviders and passed preloadedState with userRole - admin and initialEntries - '/admin', but in such case only empty div is rendered after AdminNavBar. Outlet component depends on routes and Suspence fallback is not working since Outlet is not loading. That is why I decided to add wrapping.

![image](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/17857767/14a53e88-5028-4261-bf09-7bee7787c743)

![image](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/17857767/8ad84564-5c82-4a4f-a53c-fe3ad2aa6ef7)